File: app/src/main/java/com/thesunnahrevival/sunnahassistant/views/interfaces/ReminderItemInteractionListener.java
package com.thesunnahrevival.sunnahassistant.views.interfaces;

import android.view.View;
import android.widget.CompoundButton;

import com.thesunnahrevival.sunnahassistant.data.model.Reminder;

public interface ReminderItemInteractionListener {
    void onToggleButtonClick(CompoundButton buttonView, boolean isChecked, Reminder reminder);

    void openBottomSheet(View v, Reminder reminder);
}

File: app/src/main/java/com/thesunnahrevival/sunnahassistant/views/interfaces/ReminderItemInteractionListener.kt
package com.thesunnahrevival.sunnahassistant.views.interfaces

import android.view.View
import android.widget.CompoundButton
import com.thesunnahrevival.sunnahassistant.data.model.Reminder

interface ReminderItemInteractionListener {
    fun onToggleButtonClick(buttonView: CompoundButton, isChecked: Boolean, reminder: Reminder?)
    fun openBottomSheet(v: View, reminder: Reminder?)
}